lib/rucio/client/commands/account.py

```diff
@@ -0,0 +1,240 @@
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from argparse import SUPPRESS
+from typing import TYPE_CHECKING
+
+from rucio.client.commands.bin_legacy.rucio import list_account_usage
+from rucio.client.commands.bin_legacy.rucio_admin import (
+    add_account,
+    add_account_attribute,
+    ban_account,
+    delete_account,
+    delete_account_attribute,
+    delete_limits,
+    get_limits,
+    identity_add,
+    identity_delete,
+    info_account,
+    list_account_attributes,
+    list_accounts,
+    list_identities,
+    set_limits,
+    unban_account,
+    update_account,
+)
+from rucio.client.commands.command_base import CommandBase
+
+if TYPE_CHECKING:
+    from argparse import ArgumentParser
+
+    from rucio.client.commands.utils import OperationDict
+
+
+class Account(CommandBase):
+    def module_help(self) -> str:
+        return "Methods to add or change accounts for users, groups, and services. Used to assign privileges."
+
+    def usage_example(self) -> list[str]:
+        return [
+            "$ rucio account  # List all accounts on the instance" "$ rucio account add --account user_jdoe --type USER  # Create a new user account",
+            "$ rucio account update --account user_jdoe --email jdoe@cern.ch  # Update jdoe's email",
+            "$ rucio account update --account jdoe --ban True  # Ban jdoe",
+            "$ rucio account history --account root  # Show all the usage history for the account root",
+        ]
+
+    def list_namespace(self, parser: "ArgumentParser") -> None:
+        parser.add_argument("--type", dest="account_type", default="USER", help="Account Type", choices={"USER", "GROUP", "SERVICE"})
+        parser.add_argument("-a", "--account", dest="account", help="Account name")
+        parser.add_argument("--id", dest="identity", action="store", help="Identity (e.g. DN)")
+        parser.add_argument("--filters", dest="filters", action="store", help="Filter arguments in form `key=value,another_key=next_value`")
+
+    def add_namespace(self, parser: "ArgumentParser") -> None:
+        parser.add_argument("--type", dest="accounttype", default="USER", help="Account Type (USER, GROUP, SERVICE)")
+        parser.add_argument("-a", "--account", dest="account", help="Account name")
+        parser.add_argument("--email", dest="accountemail", help="Add an email address associated with the account")
+
+    def show_namespace(self, parser: "ArgumentParser") -> None:
+        parser.add_argument("--type", dest="accounttype", default="USER", help="Account Type (USER, GROUP, SERVICE)")
```

Comment:
Forgive me, I still don’t understand this. `info_account` in the original `rucio-admin` does not take the account type into consideration (and I don’t see how it would make sense to do so).